server/api.js

```diff
@@ -14,5 +14,111 @@ router.get("/", (_, res, next) => {
 		res.json({ message: "Hello, world!" });
 	});
 });
+router.get("/graduates", (_, res, next) => {
+	
+	Connection.connect((err) => {
+		if (err) {
+			return next(err);
+		}
+		Connection.query('SELECT * FROM graduates', (error, result) => {
+				res.json(result.rows);
+			})
+	});
+});
+router.post("/graduate",(_, res, next) => {
+	const newFirstName =req.body.first_name;
+	const newSurname=req.body.surname;
+	const newCity=req.body.city;
+	const newPersonal_bio=req.body.personal_bio;
+	const newPast_experience=req.body.past_experience;
+
```

Comment:
if you have more fields you should use the destructing operator
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment